src/styles/card.scss

```diff
@@ -0,0 +1,54 @@
+.card {
+  width: $card-width;
+  padding: 20px;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  font-family: $font-family;
```

Comment:
The variable `$font-family` is used here, but it's not defined within this file. Ensure that this variable is defined in a file that is imported before this one, or define it here if it's missing.